Specs/NumberWithUnit/English/DimensionModel.json

```diff
@@ -1360,5 +1360,90 @@
         "End": 89
       }
     ]
+  },
+  {
```

Comment:
Can you add some spec cases to show that multiple temperature cases don't return false positives?

Reply:
Similarly to temperature to not catch "turn a 45 degrees angle" and similar as ambiguity filter.

Reply:
Hi - Will this be propagated to the other languages currently supporting dimensions?

Reply:
addded specs and ambiguity filters to disambiguate temperature/angle cases.
propagate solution to other languages (NL|FR|DE|IT|PT|ES).